docs/pages/versions.js

```diff
@@ -45,7 +45,7 @@ Page.getInitialProps = async () => {
    */
   const versions = [];
   branches.forEach((branch) => {
-    if (FILTERED_BRANCHES.indexOf(branch.name) === -1) {
+    if (FILTERED_BRANCHES.includes(branch.name)) {
```

Comment:
This is incorrect. This condition should be negated because the original code was using `=== -1` (element is not found in the array) and not `!== -1`.

```suggestion
    if (!FILTERED_BRANCHES.includes(branch.name)) {
```

Reply:
Oh I've missed that ! Thanks for review